File: n557-reverse-words-in-a-string-iii.go
package main

import "strings"

func reverseWords(s string) string {
	ret := []string{}

	for _, word := range strings.Split(s, " ") {
		data := []rune(word)
		reverse := []rune{}

		for i := len(data) - 1; i >= 0; i-- {
			reverse = append(reverse, data[i])
		}

		ret = append(ret, string(reverse))
	}

	return strings.Join(ret, " ")
}

File: n557-reverse-words-in-a-string-iii.js
var reverseWords = function(s) {
    var ret = []

    for (var word of s.split(' ')) {
        ret.push(word.split('').reverse().join(''))
    }

    return ret.join(' ')
}

var s = "Let's take LeetCode contest"

console.log(reverseWords(s))
